MonstieWash/Assets/Scripts/Tool.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "Tool", menuName = "ScriptableObjects/Tool")]
+public class Tool : ScriptableObject
+{
+    public Sprite mask;
+    [HideInInspector]
+    public float strength;
+    [Range(1, 10)] 
+    public int size = 1;
+    [HideInInspector] 
+    public byte[] maskPixels;
```

Comment:
Could separate what is/isn't shown in inspector

Reply:
Will do, good idea! I thought that was messy but wasn't sure the best way to lay it out.